servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/IoThreadFactory.java

```diff
@@ -72,12 +78,13 @@ public Thread newThread(Runnable r) {
         return t;
     }
 
-    private static final class AsyncContextHolderNettyThread extends FastThreadLocalThread
-            implements AsyncContextMapHolder {
+    public static final class NettyIoThread extends FastThreadLocalThread
```

Comment:
I don't think this needs to be public, and I made it package private in https://github.com/apple/servicetalk/pull/1806. wdyt?

Reply:
OK